22-ajax-lab/src/01-swapi.jsx

```diff
@@ -10,19 +10,117 @@
  * allowing a user to expand from a Person to a Film she played in,
  * to another Person from that film and so on.
  */
-
 import React from 'react';
+import $ from 'jquery';
+
+const PersonDao = {};
```

Comment:
Why like that? Why not just putting search on PersonDao like this:
```
const PersonDao = {
  search(){
     return $.getJSON(`http://swapi.co/api/people/${id}`);
  }
}
```

I also advise in your case something else. create apiService object which will deal wiht api. Something like that:

```
function getApi(baseApi = "http://swapi.co/api"){
   return {
       searchPeople: (id) => $.getJSON(`${baseApi}/people/${id}),
       searchFilms: (id) => $.getJSON(`${baseApi}/films/${id})
   }
}

```